toqito/state_props/is_unextendible_product_basis.py

```diff
@@ -39,7 +39,7 @@ def is_unextendible_product_basis(vecs: list[np.ndarray], dims: list[int]) -> tu
     >>>
     >>> non_upb_tiles = np.array([tile(i) for i in range(4)])
     >>> dims = np.array([3, 3])
-    >>> is_unextendible_product_basis(non_upb_tiles, dims)
+    >>> is_unextendible_product_basis(non_upb_tiles, dims)  # doctest: +ELLIPSIS, +NORMALIZE_WHITESPACE
```

Comment:
Hmm, are these comments needed here (i.e. the `doctest` stuff)

Reply:
Yes, otherwise the test would fail even when I made sure I copied & pasted the expected output.